server/util/responseCollectionUser.js

```diff
@@ -0,0 +1,5 @@
+const responseCollectionUser = {
+    username: 'NVDA Bot'
+};
```

Comment:
Response Collector and Collection Jobs are terms used occasionally throughout the PR, and I think they're both synonymous with automation. So `responseCollectionUser` is basically equivalent to `automationUser`, `isSupportedByResponseCollector` is basically equivalent to `isSupportedByAutomation`, and `CollectionJob` is equivalent to `AutomationJob`. Just being mindful of the friction readers might feel when they encounter new verbiage I wonder if it would be possible to replace instances of the term "response collection", "response collector" or "collection job", with equivalents using the term "automation" or "bot".

Reply:
Good call out. This was the result of differences in the system as it matured. Also, internal spec docs called the system a response collection system but all of our conversations call it automation. There are some tricky things with the term 'Automate' and 'Automation'. For ex. `CollectionJob` and `AutomationJob` are not really the same because the `CollectionJob` is what is `automated`. It is job of a collecting responses that is automated. So automation is the system of action and the collection job is what is produced and the response collector is what does the producing of collections. The distinction between automation and 'collections', 'collection jobs', and 'responses' makes since to me but it gets a little trickier with the introduction of `response collector` which is a system that is itself automated.

Anyways! I am losing myself in it a bit. I'll look for opportunities to simplify the vocabulary.

Reply:
>For ex. CollectionJob and AutomationJob are not really the same because the CollectionJob is what is automated.

I'm cool with that, I'm glad it's something we thought through. If `AutomationJob` is off the table, I think I do prefer `CollectionJob` to something like `AutomationCollectionJob` which is just too dang wordy. Seems like we can roll ahead without needing to make any big change here.